### DISCUSSION TITLE
I created a formula column using mouse clicking on notion. I dont want to create the result of this formula column when I create new rows of database. How to do that?

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [jheddings/notional#175] [Q&A]
# I created a formula column using mouse clicking on notion. I dont want to create the result of this formula column when I create new  rows of database. How to do that? 

If it is impossible to ignore columns, then what code should i add to my props dict to achieve the same goal. 
I read the mannual. I cant generate proper formula result data based on `stringFormula` or `formula` or  `FormulaResult` in `types` section. Please help, thanks.

**Comment by jheddings** [comment]
If you want to ignore the column, you simply omit that column when making updates.  Can you provide some examples of code and errors that you are seeing?

There is a known issue (#59 and #9) that may be getting in your way.  Those cases provide some workarounds if that helps.

**Reply by tyehuoPHD** [reply]
o sorry. the operation is `create new row` **not `update old row`**.

How to add right `types.XXXX` to my props dict, which will be used in the following creating new row process `notion.pages.create(parent = db, title = df.loc[i,'title'], properties = page_dict)`

**Reply by jheddings** [reply]
Ah, I see what you mean.  In this case, simply leave the formula type out of your `page_dict` when you create the new row.  It will automatically be populated according to the schema of the table for any new records.

In fact, if you try to create a new row and provide the formula, you will run into the bugs I mentioned above.  Those column types are read-only in the API, and will raise an error if you include them in the `create()` or `update()` methods of pages.

**Reply by tyehuoPHD** [reply]
Indeed. It works like a charm. thanks man.
